build/SourceLink.props

```diff
@@ -0,0 +1,11 @@
+<Project>
+  <PropertyGroup>
+    <SourceLinkCreate>true</SourceLinkCreate>
```

Comment:
Forcing it on with `<SourceLinkCreate>true</SourceLinkCreate>` is just to show that it works. The line should be removed before a merge.